src/main/java/net/atos/zac/gebruikersvoorkeuren/model/TabelInstellingen.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * SPDX-FileCopyrightText: 2023 Atos
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+package net.atos.zac.gebruikersvoorkeuren.model;
+
+import static net.atos.zac.util.FlywayIntegrator.SCHEMA;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.SequenceGenerator;
+import javax.persistence.Table;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+
+@Entity
+@Table(schema = SCHEMA, name = "tabel_instellingen")
+@SequenceGenerator(schema = SCHEMA, name = "sq_tabel_instellingen", sequenceName = "sq_tabel_instellingen", allocationSize = 1)
+public class TabelInstellingen {
+
+    /** Naam van property: {@link TabelInstellingen#medewerkerID} */
+    public static final String MEDEWERKER_ID = "medewerkerID";
+
+    /** Naam van property: {@link TabelInstellingen#lijstID} */
+    public static final String LIJST_ID = "lijstID";
+
+    public static final int DEFAULT_AANTAL_PER_PAGINA = 25;
+
+    @Id
+    @GeneratedValue(generator = "sq_tabel_instellingen", strategy = GenerationType.SEQUENCE)
+    @Column(name = "id_tabel_instellingen")
+    private Long id;
+
+    @NotNull
+    @Column(name = "id_lijst_enum", nullable = false)
+    @Enumerated(EnumType.STRING)
+    private Werklijst lijstID;
+
+    @NotBlank
+    @Column(name = "id_medewerker", nullable = false)
+    private String medewerkerID;
+
+    @NotBlank
```

Comment:
@NotNBlank is hier niet de juiste validatie. Je kunt her wel valideren dat aantalPerPagina > 0 && <= 100
Met @Positive en @Max

Reply:
Ja, had ik gezien. Komt mee in de volgende commit